doc/index.html

```diff
@@ -583,24 +584,24 @@ <h3 id="command-line-options">Command-line options</h3>
   <li id="optiongroup:Code-under-test">Code under test
     <ul>
       <li id="option:testclass"><b>--testclass=</b><i>string</i> <tt>[+]</tt>. The fully-qualified name of a class to test.
- This class is tested in addition to any specified using
- <tt>--classlist</tt>, and must be accessible from the package of the tests
+ This class is tested in addition to any specified using 
```

Comment:
What is the reason for all these whitespace changes?
There shouldn't be trailing whitespace in the source files.  However, that trailing whitespace wasn't carried over to the manual before this commit.